ahk/registery.py

```diff
@@ -0,0 +1,117 @@
+from ahk.script import ScriptEngine
+import os
+
+
+class RegisteryMixin(ScriptEngine):
+
+    def _render_template(self, template_name, *args, **kwargs):
+        return self.render_template(
+            os.path.join("registery", template_name),
+        )
+
+    def _run_template(self, template_name, *args, **kwargs):
```

Comment:
I like what you've done here. Good way to make the rest of the code a little more concise.